### PR TITLE
Pytest for send_job

### PR DESCRIPTION
**Pull request by egenc**
[Jira ticket](https://lifebit.atlassian.net/browse/CLOUDOSCLI-107?atlOrigin=eyJpIjoiYzBlZTYwY2RiZjk4NGYxOWE1Mjc0ZWY2NGI4YzVmMjciLCJwIjoiaiJ9)
Purpose of this PR is to add tests for the function `Job.send_job` in CloudOS.

Test data:
```
tests/test_data/process_job_list_initial_json.json
tests/test_data/projects.json
tests/test_data/workflows.json
```
Tests are made for checking if the function `send_job` working as intended

To run the test:
```
python3 -m pytest -s -v
```
Expected output:
image

### CODE REVIEW

**Comment by danielboloc** [comment]
@egenc looks good! Could you fix the conflicts before I can approve?

**Comment by egenc** [comment]
@danielboloc , I have fixed the conflict, could you please review?

File: tests/test_jobs/test_send_job.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            headers=header,
+            match=[matchers.query_param_matcher(params_job)],
+            status=200)
+    responses.add(
+            responses.GET,
+            url=f"{CLOUDOS_URL}/api/v1/projects?{search_str_pro_wf}",
+            body=create_json_project,
+            headers=header,
+            match=[matchers.query_param_matcher(params_pro_wf)],
+            status=201)
```
Comment (danielboloc) [inline tests/test_jobs/test_send_job.py:56]:
```suggestion
            status=200)
```

File: tests/test_jobs/test_send_job.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    responses.add(
+            responses.GET,
+            url=f"{CLOUDOS_URL}/api/v1/workflows?{search_str_pro_wf}",
+            body=create_json_workflow,
+            headers=header,
+            match=[matchers.query_param_matcher(params_pro_wf)],
+            status=202)
```
Comment (danielboloc) [inline tests/test_jobs/test_send_job.py:63]:
```suggestion
            status=200)
```

**Review by danielboloc — COMMENTED** [review]
@egenc I would leave all response status as `200`

**Comment by egenc** [comment]
thanks @danielboloc , ready to merge now.

**Review by danielboloc — APPROVED** [review]

**Pull request merged**